README.md

```diff
@@ -34,7 +34,7 @@ thus minimizing human intervention and human error.
 Status: v1, stable
 
 CosmosFullNode is the flagship CRD. Its purpose is to deploy highly-available, fault-tolerant RPC nodes. 
-It will eventually support validator sentries, persistent peers, and seeds, but is currently only suitable for RPC.
+It will eventually support persistent peers and seeds, but is currently only suitable for RPC and validator sentries.
```

Comment:
Is the reason we don't "support" persistent peers and seeds because the node keys are only persistent as to the PVC?

Reply:
Ideally, I'd like the entire deployment to masquerade as a single peer. But we need to test what happens if we load balance across pods with different heights or state.